.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -107,7 +106,7 @@ jobs:
             tunnel: 'vxlan'
             encryption: 'ipsec'
             encryption-node: 'false'
-            endpoint-routes: 'true'
+            endpoint-routes: 'false' # Due to https://github.com/cilium/cilium/pull/22333
```

Comment:
Should this be in its own commit? Or in the first commit with a backporter note?

Reply:
:+1: Added the note in the first commit. Same for the https://github.com/cilium/cilium/pull/27174 PR.